scalalib/test/src/HelloWorldTests.scala

```diff
@@ -169,7 +169,7 @@ object HelloWorldTests extends TestSuite {
 
   object HelloWorldOnlyDocVersion extends HelloBase {
     object core extends HelloWorldModule {
-      def scalacOptions = T(Seq("-Ywarn-unused", "-Xfatal-warnings"))
+      def scalacOptions = T(Seq("-Ywarn-unused"))
```

Comment:
Now, docs trigger compilation and in case of `HelloWorld` it fails with a warning (turn to error) about unused import.